src/components/views/admin/BuilderView.tsx

```diff
@@ -168,12 +170,25 @@ const BuilderView: FC = () => {
       pr={DEFAULT_MARGIN}
     >
       <PublicAlert />
-      <Typography variant="h4" sx={{ color: '#5050d2' }}>
-        {t(TEXT_ANALYSIS.BUILDER_VIEW_TITLE)}
-      </Typography>
-      {isChanged && (
+      <Stack direction="row" spacing={2} alignItems="center">
+        <Typography variant="h4" sx={{ color: '#5050d2' }}>
```

Comment:
use `color="primary"` ?

Reply:
The theme was not configured correctly in this app. I will open an issue for that.